### PR TITLE
Add ability to open all visible URLs in browser

### PR DESCRIPTION
**Pull request by asmorodskyi**
Add ability to open all visible URLs in browser

### CODE REVIEW

File: cmd/openqa-revtui/openqa-revtui.go [hunk 1]
```diff
@@ -598,6 +599,17 @@ func tui_main(tui *TUI, instance gopenqa.Instance) error {
 			// Shift through the sorting mechanism
 			tui.SetSorting((tui.Sorting() + 1) % 2)
 			tui.Update()
+		} else if key == 'o' {
+			for _, job := range tui.Model.jobs {
+				if !tui.hideJob(job) {
+					var err error
+					err = exec.Command("xdg-open", job.Link).Start()
```
Comment (grisu48) [inline cmd/openqa-revtui/openqa-revtui.go]:
```suggestion
					err := exec.Command("xdg-open", job.Link).Start()
```

Let's stay go idomatic :slightly_smiling_face: (And please remove the previous line)

File: cmd/openqa-revtui/openqa-revtui.go [hunk 2]
```diff
@@ -598,6 +599,17 @@ func tui_main(tui *TUI, instance gopenqa.Instance) error {
...
+					if err != nil {
+						tui.SetStatus(fmt.Sprintf("Error: %s", err))
```
Comment (grisu48) [inline cmd/openqa-revtui/openqa-revtui.go:607]:
```suggestion
						tui.SetStatus(fmt.Sprintf("Error: %s", err))
						tui.Update()
						return
```

In the error case it likely will fail for all subsequent calls as well, so it makes little sense to iterate the same error x times.

File: cmd/openqa-revtui/tui.go [hunk 3]
```diff
@@ -436,7 +436,7 @@ func (tui *TUI) buildHeader(width int) []string {
 	lines := make([]string, 0)
 	if tui.header != "" {
 		lines = append(lines, tui.header)
-		lines = append(lines, "q:Quit   r:Refresh   h:Hide/Show jobs   m:Toggle RabbitMQ tracker   s:Switch sorting    Arrows:Move up/down")
+		lines = append(lines, "q:Quit   r:Refresh   h:Hide/Show jobs o: Open in browser  m:Toggle RabbitMQ tracker   s:Switch sorting    Arrows:Move up/down")
```
Comment (grisu48) [inline cmd/openqa-revtui/tui.go]:
```suggestion
		lines = append(lines, "q:Quit    r:Refresh    h:Hide/Show jobs    o:Open links    m:Toggle RabbitMQ tracker   s:Switch sorting    Arrows:Move up/down")
```

I think "Open links" is more descriptive than "Open in browser", and let's keep the spacing between the elements consistent

**Review by grisu48 — COMMENTED** [review]
Thanks for the contribution :slightly_smiling_face: 

I just have some minor nitpicks, overall LGTM!

File: cmd/openqa-revtui/openqa-revtui.go (hunk 1 above)
Comment (asmorodskyi) [reply cmd/openqa-revtui/openqa-revtui.go]: done

File: cmd/openqa-revtui/openqa-revtui.go (hunk 2 above)
Comment (asmorodskyi) [reply cmd/openqa-revtui/openqa-revtui.go:607]: generally agree but I have different idea how to achieve the same . Please check my version if you don't like it I will switch to yours

File: cmd/openqa-revtui/tui.go (hunk 3 above)
Comment (asmorodskyi) [reply cmd/openqa-revtui/tui.go]: done

File: cmd/openqa-revtui/openqa-revtui.go (hunk 2 above)
Comment (grisu48) [reply cmd/openqa-revtui/openqa-revtui.go:607]: Jep, a `break` also works nicely.

**Review by grisu48 — APPROVED** [review]

**Comment by grisu48** [comment]
Thanks for the contribution!

**Pull request merged**